doc/manual/config-advanced.rst

```diff
@@ -16,30 +16,25 @@ path of your installation) as follows:
 - *Affiliation logos*: these will be shown with the teams that are
   part of the affiliation, if the ``show_affiliation_logos`` configuration
   option is enabled. They can be placed in
-  `public/images/affiliations/1234.png` where *1234* is the numeric ID
+  `public/images/affiliations/1234.png` where *1234* is the external ID
```

Comment:
Should we explain what external means at the first mention or somewhere?

Reply:
That would be in import.rst. What do you suggest to add?

Reply:
I added a block to the manual and linked to it where we mention it.